### PR TITLE
Report on issue #1078

### PR DESCRIPTION
**Pull request by shark-S**

### CODE REVIEW

**Comment by shark-S** [comment]
@ajdapretnar guide me if Report on issue #1078 have some error.

**Comment by ajdapretnar** [comment]
Stochastic Gradient is not even available when I'm testing your implementation. Something is not quite right....

File: Orange/widgets/regression/owsgdregression.py [hunk 1]
```diff
@@ -197,6 +197,16 @@ def _on_lrate_changed(self):
...
 
+    def send_report(self):
+        """ report learner name,model and data"""
+        self.report_items((("Name",self.learner_name),))
+        self.report_items("Model Parameters",(("Loss function to be used ",self.loss_function)
+                                            ,("Penalty ",self.penalty_type)
+                                            ,("Learning rate",self.learning_rate)
+                                            ,("Constant",self.constant))
+        if self.data:
+            self.report_items("Data",self.data)
```
Comment (kernc) [inline Orange/widgets/regression/owsgdregression.py]: Surely you don't need [6 commits](https://github.com/biolab/orange3/pull/1098/commits) for these couple of lines?

File: Orange/widgets/regression/owsgdregression.py [hunk 2]
```diff
@@ -197,6 +197,16 @@ def _on_lrate_changed(self):
         for spin, enabled in zip(self._lrate_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        """ report learner name,model and data"""
+        self.report_items((("Name",self.learner_name),))
+        self.report_items("Model Parameters",(("Loss function to be used ",self.loss_function)
+                                            ,("Penalty ",self.penalty_type)
+                                            ,("Learning rate",self.learning_rate)
+                                            ,("Constant",self.constant))
```
Comment (kernc) [inline Orange/widgets/regression/owsgdregression.py]: Please make this PEP-8 conformant (space after comma) and indented correctly (by one more space character). It's important to strive to leave code in better shape than you get it, not worse. :smiley:

**Comment by shark-S** [comment]
actually they are previous issues commits i have forget them to delete next time i will take care of it .

**Comment by kernc** [comment]
You can merge the commits into one by [squashing them](https://www.google.com/search?q=github+squash+commits).
